### PR TITLE
fix(cart): fix cart in-memory service

### PR DESCRIPTION
**Pull request by lderrickable**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: libs/cart/testing/src/in-memory-backend/cart/cart.service.ts [hunk 1]
```diff
@@ -27,7 +27,8 @@ export class DaffInMemoryBackendCartService implements InMemoryDbService {
   post(reqInfo: RequestInfo) {
     return reqInfo.utils.createResponse$(() => {
       let body;
-      const action = this.getAction(reqInfo);
+			const action = this.getAction(reqInfo);
+			console.log(action);
```
Comment (damienwebdev) [inline libs/cart/testing/src/in-memory-backend/cart/cart.service.ts]: Log

Comment (griest024) [inline libs/cart/testing/src/in-memory-backend/cart/cart.service.ts]: the log should be removed

Comment (lderrickable) [reply libs/cart/testing/src/in-memory-backend/cart/cart.service.ts]: removed

Comment (lderrickable) [reply libs/cart/testing/src/in-memory-backend/cart/cart.service.ts]: removed

**Pull request merged**